workspaces/adventure-pack/goodies/typescript/Math.gcd/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Math.gcd", () => {
+  it.each([
+    [48, 18, 6],
+    [17, 13, 1],
+    [36, 60, 12],
+    [5, 0, 5],
+    [0, 5, 5],
+    [-5, 10, 5],
+    [-7, 3, 1],
+    [-8, -12, 4],
+    [0, 0, 0],
```

Comment:
Fwiw, the greatest common divisor of 0 and 0 is not well-defined, and an argument could be made that this should return Infinity, or throw an error. But I don't think it's worth having a special case in the code just for that, so this is fine. We can let the user defend against this if need be.